File: ios/Khrysalis/com/lightningkite/khrysalis/observables/binding/SelectMultipleDatesView.binding.actual.swift
//Stub file made with Khrysalis 2 (by Lightning Kite)
import Foundation


//--- SelectMultipleDatesView.bind(MutableObservableProperty<Set<Date>>)
public extension SelectMultipleDatesView {
    func bind(_ dates: MutableObservableProperty<Set<Date>>) -> Void {
        self.dates = dates
    }
    func bind(dates: MutableObservableProperty<Set<Date>>) -> Void {
        return bind(dates)
    }
}

File: android/src/main/java/com/lightningkite/khrysalis/observables/binding/SelectMultipleDatesView.binding.actual.kt
package com.lightningkite.khrysalis.observables.binding

import com.lightningkite.khrysalis.observables.*
import com.lightningkite.khrysalis.views.android.SelectDayView
import com.lightningkite.khrysalis.views.android.SelectMultipleDatesView
import java.util.*

fun SelectMultipleDatesView.bind(dates: MutableObservableProperty<Set<Date>>) {
    this.dates = dates.transformed(
        read = { it.map { Calendar.getInstance().apply { timeInMillis = it.time } }.toSet() },
        write = { it.map { it.time }.toSet() }
    )
}
